src/dashboard/public/styles.css

```
@@ -601,6 +601,27 @@ body {
+    background: var(--surface-strong);
+    color: var(--text-dim);
+    font-size: 0.85rem;
+    line-height: 1.5;
+}
+.mcp-note code { font-size: 0.82rem; }
+.mcp-note-warn {
+    background: var(--warn-bg, #fffbe6);
+    border-color: var(--warn-border, #ffe58f);
+    color: var(--warn-text, #7a5f00);
+}
+.mcp-test-result { font-size: 0.78rem; margin-top: 0.35rem; color: var(--text-mute); word-break: break-word; }
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)